robotframework-ls/src/robotframework_ls/impl/ast_utils.py

```diff
@@ -1,17 +1,20 @@
 import sys
 from typing import Iterator, Optional, List, Tuple, Any, Union
-
+import re
 import ast as ast_module
+from robot.api import Token
```

Comment:
imports from `robot` should always be local imports, not top-level. 

-- note: in general, anything that isn't super-common should be a local import (in this case, the other imports are common, so, that's ok), and in particular, importing `robot` is super-slow, so, it should be done only when really needed.